.github/workflows/wizard-py-lint.yaml

```diff
@@ -0,0 +1,21 @@
+name: Python AST CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  my-job:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: CI build
+      env:
+        TEST_ENV: test
+        TRAMPOLINE_BUILD_FILE: ci-cd/lint_wizard_py.sh
+        TRAMPOLINE_DOCKERFILE: ci-cd/wizard_py_dockerfile
+        TRAMPOLINE_IMAGE: gcr.io/cloud-devrel-public-resources/docker-ci-helper/python
```

Comment:
You're setting `TRAMPOLINE_DOCKERFILE`.

This means you want to build the docker image for each test. Is this what you really want?

Reply:
Removed.